.eslintrc.cjs

```diff
@@ -7,6 +7,57 @@ const path = require('path');
 const rulesDirPlugin = require('eslint-plugin-rulesdir');
 rulesDirPlugin.RULES_DIR = path.join(__dirname, 'eslint');
 
+const dprintConfig = {
```

Comment:
Would it make sense to drop things that are the default anyway?
I was running dprint with:
```json
  "typescript": {
    "lineWidth": 100,
    "indentWidth": 2,
    "useTabs": false,
    "semiColons": "prefer",
    "quoteStyle":"alwaysSingle",
    "newLineKind": "crlf",
    "useBraces": "maintain",
    "bracePosition": "maintain",
    "singleBodyPosition": "maintain",
    "nextControlFlowPosition": "maintain",
    "trailingCommas": "onlyMultiLine",
    "operatorPosition": "maintain"
  },
  ```
From there, I think `useTabs`, `semiColons`, `singleBodyPosition`, and `trailingCommas` are all defaults and could be dropped as well.

Notably I also don't see `bracePosition` in your settings which may be related to the raidemulator stuff.

Reply:
Thanks, done!